tests/test_basic.py

```diff
@@ -240,13 +240,15 @@ def test_add_storage(rook_cluster):
 
     # wait for the additional osd
     # this may take a while
-    while osds_expected != osds_new:
+    while osds == osds_new:
         if i == 60:
             pytest.fail("rook did not add an additional osd-node")
```

Comment:
Instead, we could just log here what the current details are if we're failing for some reason.

Reply:
no... if so we can not detect the situation when there are more osds expected than required... like we have no, because of previous test interrupted early before waiting of the new node getting online and new osds appeared